thoth/user_api/api_v1.py

```diff
@@ -270,7 +270,7 @@ def post_advise_python(
     )
     github_webhook_params_present = sum(github_webhook_params)
     if github_webhook_params_present != 0 and github_webhook_params_present != len(github_webhook_params):
-        return {"parameters": parameters, "error": "Not all webhook parameters provided for GitHub webhook"}, 400
+        _LOGGER.info("Not all webhook parameters provided for GitHub webhook")
```

Comment:
Could you verify which parameters are set? The condition looks good to me.

I suspect the origin parameter is supplied - that causes the check to return the error.

```python
    if github_webhook_params_present != 0 and github_webhook_params_present != len(github_webhook_params) and not origin:
        return {"parameters": parameters, "error": "Not all webhook parameters provided for GitHub webhook"}, 400
```

Reply:
... and remove `origin` from `github_webhook_params`

Reply:
I guess if we remove origin from the check, it should solve it.

Reply:
+1

however, it needs to be part of the check - we expect it to be present if all the rest webhook params are provided `(github_webhook_params_present != len(github_webhook_params) or not origin)`

Reply:
That makes sense! Thanks.